### ISSUE TITLE
[Bug]: resolveMath is including / of typography shorthand in calculation

### ISSUE THREAD

**Issue by SarahFieldingSmith** [tokens-studio/sd-transforms#106]
### What happened?

Hi,

I'm experiencing an issue where resolveMath is including the '/' of the typography shorthand in the calculation.

Here is the json:
        
```
        "semibold": {
          "value": {
            "fontFamily": "Open Sans",
            "fontWeight": "600",
            "fontSize": "16",
            "lineHeight": "16 * 1.5"
          },
          "type": "typography"
        },
```
Here is the output with resolveMath:

`--token-semantic-typography-body-md-semibold: 600 1.5px Open Sans;
`        
and the output without resolveMath:
 
`--token-semantic-typography-body-md-semibold: 600 16px/16 * 1.5 Open Sans;
`
Whilst I would expect:

`--token-semantic-typography-body-md-semibold: 600 16px/24px Open Sans;
`
This was working previously in version 5, but we have updated today to make use of a new bug fix, and noticed this regression.

Would appreciate any help.

### Version

0.8.1

**Comment by jorenbroekema** [comment]
Reproduced it. I'm gonna take a look at it asap.

**Comment by jorenbroekema** [comment]
Another note btw:
https://github.com/tokens-studio/sd-transforms#full-example see this full example, when the transforms are used as standalone, directly, and not by using the transformGroup, the order actually matters a lot. If you see that your typography CSS shorthand is being resolved as a math expression, it is because the `ts/resolveMath` transform is running **after** the `ts/typography/css/shorthand`, if you flip the order to the correct order, it shouldn't happen anymore.

Math expressions such as your lineheight (`16 * 1.5`) not being resolved properly, is fixed here https://github.com/tokens-studio/sd-transforms/pull/107 , to be released as `0.8.2`
